src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class FigureProducer {
+
+    private static final int RANDOM_NUMBER = 5;
+    private static String [] figureArr = { "SQUARE", "RECTANGLE", "RIGHT TRIANGLE",
+            "CIRCLE", "ISOSCELES TRAPEZOID" };
```

Comment:
let's make this a constant as well
```suggestion
    private static final String [] FIGURES = { "SQUARE", "RECTANGLE", "RIGHT TRIANGLE",
            "CIRCLE", "ISOSCELES TRAPEZOID" };
```